README.md

```
@@ -35,6 +36,26 @@ This script copies the installation command for LinuxDroid to your clipboard, ma
    - NethunterRootLess (Hacking specialized Linux)<img src="https://github.com/AryanVBW/Myimage/releases/download/m3/kali-nethunter-dragon-tm.png" height="12">
    - BlackBox(coming🔜)<img src="https://raw.githubusercontent.com/AryanVBW/LinuxDroid/main/LinuxDroid/Icon/bosckbocBg.png" height="12">
    - CentOS(coming 🔜)<img src="https://raw.githubusercontent.com/AryanVBW/LinuxDroid/main/LinuxDroid/Icon/centosbg.png" height="12">
+
+---
+### Linux Distribution Comparison
+
+Quick look at RAM, storage, and desktop environments for each distro.
+
+| Distribution | RAM | Storage | Desktop Environments | Best For |
+|--------------|-----|---------|----------------------|----------|
+| <img src="https://raw.githubusercontent.com/AryanVBW/LinuxDroid/main/LinuxDroid/Icon/linux-mint.svg" height="16"> **Linux Mint** | 2 GB | 15 GB | Cinnamon, MATE, XFCE | Beginners, everyday use |
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)